### DISCUSSION TITLE
How to change default of 10 rows for Datatable?

### DISCUSSION THREAD

**Discussion by flairzhealth** [datapane/datapane#88] [Help] [answered]
New to datpane API and exploring features. We need a way to set the limit to 50 rows.  How can it be done ?

**Comment by mands** [comment]
Hi there - unfortunately that's not currently possible - at the moment the table is limited to showing only 10 rows at a time, but you can view the other rows by pressing the Next/Prev buttons.

We can add support for increasing the page size, that would be a useful feature, and we're also looking an introducing a more advanced table viewer. Hope that helps!

**Reply by Davidyang1989** [reply]
more rows in one page would be very useful, hope can be available in short time!

**Reply by mands** [reply]
We're just releasing a revamp of the DataTable component that supports scrolling like a spreadsheet and better filters and sorting - please stay tuned!

**Comment by mands** [comment, marked as answer]
New Datatable component has been released that provides a spreadsheet-type view with all rows - hope that helps!

**Reply by flairzhealth** [reply]
Thank you,  will explore this feature..
